src/test/regress/spec/isolation_logical_replication_nonsu_nonbypassrls.spec

```diff
@@ -0,0 +1,134 @@
+// isolation_logical_replication_nonsu_nonbypassrls
+// test moving a single shard that has rls
+// owned by a user that is neither superuser nor bypassrls
+// PG15 added extra permission checks within logical replication
+// this test makes sure that target table owners should still
+// be able to replicate despite RLS policies.
+// Relevant PG commit: a2ab9c06ea15fbcb2bfde570986a06b37f52bcca
```

Comment:
maybe add 1 line comment about using sequential mode:

`setup mode involves lots of DDL inside a single tx block, so use sequential mode`